# 16s rRNA sequencing

Description:

The 16S rRNA gene is used for phylogenetic studies as it is highly conserved between different species of bacteria and archaea. Carl Woese pioneered this use of 16S rRNA. It is suggested that 16S rRNA gene can be used as a reliable molecular clock because 16S rRNA sequences from distantly related bacterial lineages are shown to have similar functionalities. Some thermophilic archaea (e.g. order Thermoproteales) contain 16S rRNA gene introns that are located in highly conserved regions and can impact the annealing of "universal" primers. Mitochondrial and chloroplastic rRNA are also amplified.

In addition to highly conserved primer binding sites, 16S rRNA gene sequences contain hypervariable regions that can provide species-specific signature sequences useful for identification of bacteria. As a result, 16S rRNA gene sequencing has become prevalent in medical microbiology as a rapid and cheap alternative to phenotypic methods of bacterial identification. Although it was originally used to identify bacteria, 16S sequencing was subsequently found to be capable of reclassifying bacteria into completely new species, or even genera. It has also been used to describe new species that have never been successfully cultured. With third-generation sequencing coming to many labs, simultaneous identification of thousands of 16S rRNA sequences is possible within hours, allowing metagenomic studies, for example of gut flora.

Analysing sanger sequence based 16S rRNA for microbiome is time consuming process. So I am currently developing an algorithm to analyse and determine the microbial species using sanger sequencing method. 
